#US4R-686: us4R model R-2023 User Manual

beatawitek opened the pull request:
Please review and make the latest version of the us4R user manual available online.
Preferable ver. number is v6 (0.6; 0.06, ...)

beatawitek committed:
initial structure changes

Standardization of folder names and contents (similar to us4R-lite) #1

beatawitek committed:
Update safety.md

beatawitek committed:
structure updated, most of the text ready

beatawitek committed:
ready for review

    - Instruction updated to include the R-2023 model.
    - Document structure adapted for the online version.

pjarosik committed:
Automatically generate copyright footer.

pjarosik committed:
Minor fix

pjarosik commented on docs/content/2_hardware.md:
```diff
@@ -1,46 +1,164 @@
+*   - Model
+    - Options
+    - External Interface
+*   - R-2021 <span style="color:red">(EOL)</span>
```
@beatawitek please note that the EOL will be in red only for the web-based version of documentation

beatawitek replied:
ok, thanks for the aware

pjarosik commented on docs/content/5_guidelines-for-use.md:
````diff
@@ -1,89 +1,131 @@
+The ready to use setup consists of the **us4R™** device, a host PC computer ({numref}`us4r+pc+conn`), an LCD monitor and a set of cables.
+
+```{figure} img/us4r+pc+conn.png
+:name: us4r+pc
````
@beatawitek: this label duplicates this one: https://github.com/us4useu/us4r-user-docs/pull/2/files#diff-31ef5a303ecd1f0b752f1a20abab065425fcdec352e94d232c2ec4aebdab3d85R17

beatawitek replied:
thx, it clarifies a lot!

pjarosik commented on docs/content/intended-use_OUT.md:
@beatawitek what is the purpose of keeping these _OUT files?

beatawitek replied:
hmm 'just in case of change our minds'... no purpose... I will remove them.

pjarosik reviewed (approved):
Only some minor comments.

Please be aware that I only skimmed through most of the text (I focused only on the software part). Perhaps the document should be fully read again by @heymarcin

beatawitek committed:
HF intead of 80MSPS in spec

6.1 spec table updated with 16bit @120MSPS instead of 12bit @80 MSPS;
limitations updated accordingly

beatawitek committed:
Update 6_tech-spec.md

beatawitek committed:
Update 5_guidelines-for-use.md

beatawitek committed:
_OUT files removed

heymarcin reviewed (approved):
OK
